package.json

```diff
@@ -28,7 +28,7 @@
     "styled-components": "^5.1.0"
   },
   "devDependencies": {
-    "@etvas/etvaskit": "^1.1.32",
+    "@etvas/etvaskit": "^1.1.8",
```

Comment:
This looks odd ... the version is not correct

Reply:
1dc4eecbba23b08655bfc5e9cd52112f6c35e7ba